Assets/_BForBoss/_Core/Scripts/Boss/DerekHealthView.cs

```diff
@@ -25,22 +25,11 @@ public void SetState(DerekHealthViewState state)
         {
             _invincibleView.gameObject.SetActive(state.IsInvulnerable);
             _dynamicView.fillAmount = state.HealthPercentage;
-            _dynamicView.color = MapHealthToColor(state.HealthPercentage);
         }
 
-        private Color MapHealthToColor(float health)
+        public void SetHealthBarColor(Color healthBarColor)
```

Comment:
Technically the view should only have 1 public method. 
`SetState()`. 

So if you want to Set the color of the view, you should put it in the state as well.